controller/handler/dashboard.js

```diff
@@ -15,41 +15,83 @@ const database = require('../../model');
  * @returns {View} Rendered page
  */
 function dashboardView (request, reply) {
-    const trial = database.sequelize.model('trial');
+    const todayDate = moment();
```

Comment:
the todayDate variable can be renamed as "currentDate" if this variable is not used elsewhere in the file